syntax = "proto3";

package ik_Gate;

message GateRequest
{
    bytes type = 1;
    bytes request_msg = 2;
}

message GateResponse
{
    bytes type = 1;
    bytes response_msg = 2;
}